dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunction.cs

```diff
@@ -432,7 +433,12 @@ await using (enumerator.ConfigureAwait(false))
         // visible to a consumer if that's needed.
         if (ex is OperationCanceledException cancelEx)
```

Comment:
Since RestApiOperationRunner is always used with functions, does it make sense to just wrap the OperationCancelledException with a KernalFunctionCancelledException in RestApiOperationRunner?

Reply:
In `RestApiOperationRunner` we don't have all of the data  to construct a `KernalFunctionCancelledException`.